org.eclipse.xtext/src/org/eclipse/xtext/workspace/IProjectConfig.java

```diff
@@ -37,4 +37,16 @@
 	 * @return the workspace config
 	 */
 	IWorkspaceConfig getWorkspaceConfig();
+
+	/**
+	 * During the build, projects are indexed, validated and compiled. The return of this method configures whether this
+	 * project is validated and compiled or not.
+	 * 
+	 * @return true iff this project is indexed only
```

Comment:
typo?

Reply:
AFAIK, 'iff' with two 'f' means 'if and only if' which would mean that this condition is complete, or to put it this way, that all other cases always return false (whereas 'if' would could be incomplete in that sense). Anyway, I can remove the second 'f' to avoid confusion.